wled00/FX.cpp

```diff
@@ -2400,9 +2400,10 @@ uint16_t ripple_base()
 
       #ifndef WLED_DISABLE_2D
       if (SEGMENT.is2D()) {
+        propI /= 2;
         uint16_t cx = rippleorigin >> 8;
         uint16_t cy = rippleorigin & 0xFF;
-        uint8_t mag = scale8(cubicwave8((propF>>2)), amp);
+        uint8_t mag = scale8(sin8((propF>>2)), amp);
         if (propI > 0) SEGMENT.draw_circle(cx, cy, propI, color_blend(SEGMENT.getPixelColorXY(cx + propI, cy), col, mag));
```

Comment:
What's the difference between cubicwave8() and sin8()?

Reply:
As per documentation cubicwave is similar to sin but has a faster rise and stays ~1 longer (closer to square wave).
It seems to me that behaviour is better with sin in 2D variant (less flickering).